terraform/terraform-remote-state/variables.tf

```diff
@@ -0,0 +1,6 @@
+# Shortnames for regions can be found here:
+# https://github.com/claranet/terraform-azurerm-regions/blob/master/REGIONS.md
```

Comment:
I notice this being repeated in the other `variables.tf` file as well. I wonder if this should be documented in a centralised location instead.